src/Integration.Vsix.UnitTests/Commands/HelpMenu/AboutDialogTests.cs

```diff
@@ -1,28 +0,0 @@
-﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
-using Moq;
-using SonarLint.VisualStudio.Integration.Vsix.Commands;
-using SonarLint.VisualStudio.Core;
-using System.Windows.Navigation;
-
-namespace SonarLint.VisualStudio.Integration.UnitTests.Commands.HelpMenu
-{
-    [TestClass]
-    internal class AboutDialogTests
```

Comment:
This class was Internal, so the build server wasn't actually picking it up. When I changed it to public, the build started to fail because we're creating a real AboutDialog and InitializeComponent is called and fails due to missing VS stuff.

Putting code directly in the xaml is a bad practice because of it but since it's one line, it's keep it as is for now and I deleted the test. If we want to test it, we need to take it out of the xaml.cs